### ISSUE TITLE
Replace the registration feature

### ISSUE THREAD

**Issue by henrinie** [Signbank/FinSL-signbank#177]
We decided to remove the registration feature.
It is not convenient for us to have users register accounts themselves, because someone has to provide these new users with the correct user rights anyway.
To be able to update the registration feature (package) to the latest version of 'django-registration' we would have needed to remove the registration from the source code anyway, as it is directly integrated into Signbanks code, and then add the package to the requirements.txt. See #132 for more information.

**Comment by henrinie** [comment]
Perhaps we should reconsider this.
If users don't register themselves, it is harder to make them change their password later.
So I think that we should just use the app 'django-registration' and remove it the apps old source code from our app.

**Comment by henrinie** [comment]
The signbank.registration app has now been removed from the project and been replaced with django-registration [https://github.com/macropin/django-registration](https://github.com/macropin/django-registration).

The whole registration / signing in flow has been redone. I replaced the log out button with a dropdown button that now has two links, *Sign out* and *Change password*. Hopefully this encourages users to change their password. Other functionality might be added later.

Still need to hunt all the remains of signbank.registration from within the codebase.
